src/components/modals/Modal.vue

```diff
@@ -3,11 +3,17 @@
     <div class="modal-backdrop" @click="close">
       <div
         class="modal"
+        v-bind:class="[isMaximized ? 'maximized' : 'not-maximized']"
```

Comment:
```suggestion
        :class="{ maximized: isMaximized }"
```
The default state should be "not maximized", so any maximization rules should be applied with the .maximized class only.

Reply:
The default state is "not maximized" (except for the reference modal which overrides it) - I'm not sure what you mean here.

Reply:
Oh, I think I got it. You want the .not-maximized class to not exist.

Reply:
I removed it and it worked fine if you move the .not-maximized rules into .modal.